### ISSUE TITLE
Need a way of explictly turning off order_by.

### ISSUE THREAD

**Issue by lloydtabb** [malloydata/malloy#1089]
It turns out that in BigQuery, in order to partition data in a result table using 'CREATE TABLE AS SELECT" the query cannot contain an ORDER BY

```
Result of ORDER BY queries cannot be partitioned by field 'flight_date'

----------Generated SQL----------

1| -- connection: bigquery
2| -- import: "flights.malloy"
3| 
4| create or replace table scratch.faa_session 
5| partition by flight_date
6| as (
7|   WITH __stage0 AS (
8|   SELECT
9|     group_set,
10|     CASE WHEN group_set IN (0,1,2) THEN
11|       SAFE_CAST(flights.dep_time  AS date)
12|       END as flight_date__0,
13|     CASE WHEN group_set IN (0,1,2) THEN
```

**Comment by carlineng** [comment]
I hit a similar issue while playing with the new `msql` functionality. While trying to materialize a large resultset using a `CREATE TABLE` statement, I got a "exceeded memory limit" error from BigQuery because of the `ORDER BY` clause that we insert into the query by default.

If we implement tagging, and that's how tables get materialized going forward, we'll definitely want to remove the `order by` in any queries used in `CREATE TABLE` queries, or else a meaningful number of them are going to fail.

**Comment by carlineng** [comment]
my bad ideas:

* `order_by: false`
* `order_by: null`
* `order_by: <constant>` -- any constant expression in the `order_by` removes the clause, and by convention we say `false`

`order_by: nothing` isn't bad, if you don't mind adding another keyword to the language.

**Comment by christopherswenson** [comment]
I prefer `null` to `nothing`.

My one worry with "order by any constant is the same as order by nothing" is that someone will try `order_by: 1` and be confused that that orders by the first field. On that note, I pretty strongly feel like we should get rid of the `order_by: 1` syntax. It feels like doing something the way SQL does it for the sake of it, not because it's good. Queries are much more readable when you write out the name of the field being ordered by.

**Comment by christopherswenson** [comment]
I also like `order_by: false`. We may want to support multiple of these.

**Comment by carlineng** [comment]
Good point on the `order_by: 1`. I don't love the way its used in SQL now, but its so widespread I think its a case where we just have to live with it.

**Comment by carlineng** [comment]
I think the most common example is:

`ORDER BY 1,2,3,4,5` where a user wants to lexically order dimension values. In cases where its a measure, i think it's more common to use `ORDER BY measure_name DESC`
